integr8ly/integreatly-operator
# [WIP] MGDAPI-4116 - modify script to build from existing index

# Issue link
[MGDAPI-4116](https://issues.redhat.com/browse/MGDAPI-4116)

# What
Add `BUILD_FRESH` parameter to build from existing index referenced by products yaml.

# Verification steps
Can successfully build an index with a new bundle:
- Add the following to `bundles/3scale-operator/bundles.yaml`
```yaml
  - name: 3scale-operator.v0.8.4
    image: quay.io/acatterm/3scale-bundle:v0.8.4
```
- Build new index with `VERSION=0.8.4 ORG=<your-org> make create/3scale/index`
- Should have new image created `quay.io/<your-org>/3scale-index:v0.8.4`
